packages/fxa-payments-server/Dockerfile

```diff
@@ -1,27 +1,37 @@
-FROM node:12-stretch AS node-builder
+FROM node:12-stretch-slim AS node-builder
 RUN groupadd --gid 10001 app  && \
     useradd --uid 10001 --gid 10001 --home /app --create-home app
 RUN mkdir /fxa-content-server && chown -R app:app /fxa-content-server
+RUN apt-get -y update && apt-get -y install git && apt-get clean && rm -rf /var/lib/apt/lists
```

Comment:
I think if you revert the first line of this docker file back to `node:12-stretch` you don't need to install `git` because it already comes with git:

```
$ docker run --rm node:12-stretch git --version
git version 2.11.0
```

Reply:
Thanks! Yeah, I mistakenly changed the first one from stretch-12 to stretch-12-slim, when I was supposed to be changing the second one :-P